src/main/java/com/epam/esm/entity/Identifiable.java

```diff
@@ -1,7 +1,17 @@
 package com.epam.esm.entity;
 
+/**
+ * This interface specifies that an object of a class implements this interface has an id field
+ *
+ * @author Alexey Bliznichenko
+ */
 public interface Identifiable {
```

Comment:
Once again: why do we need the interface?

Reply:
The main purpose of the interface is to mark entities. This is just interface-marker

Reply:
@a1exanddrovich all marker interfaces have their purposes, they're used somewhere eventually. So do we need this interface?
+ 1 question for demo to discuss marker interfaces

Reply:
@a1exanddrovich has this "kind of check" been implemented somewhere already? If not, please read about the YAGNI software development principle